inputFiles/poromechanics/StabilizedPoroStaircase_initTemp.txt

```diff
@@ -0,0 +1,2 @@
+300
```

Comment:
Are these table files unique, or can we point to existing files for an unstabilized version of the PoroStaircase problem?  Just want to avoid lots of near duplicate material.

Reply:
Since the PR was open, a similar integrated test has been merged into `develop`. So, Ryan's PR is now lighter and only adds stabilization to an existing integrated test. These tables have been removed from the PR.